catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/ABSProperties.java

```diff
@@ -22,8 +22,8 @@
 public class ABSProperties {
```

Comment:
could you remove `ABSProperties`, use the properties defined in `AzureProperties` instead?